wiki/Article_Style_Guide/en.md

```diff
@@ -68,6 +79,22 @@ The title of an article is the name of the folder that the article is located in
 For English article titles, if you need to reword the title, you must rename the folder to match the article title.
 
 All article titles are to be using the level 1 heading.
+For example:
+
+```
+# Folder Name
+```
+
+In the case where a title cannot be translated, use the language title (in English) followed by the language name (in that language) in round brackets.
+For example:
+
+```
+# osu!mania (Español)
+# Live Streaming (日本語)
```

Comment:
These examples just shows how to format the title if you **cannot** translate the title. (If you can translate the title, then just use the translation.)

~~Yes, most titles can be translated but a few (or more) articles were translated without it (or the transition from MediaWiki to Markdown got rid of them).~~